src/eliterizer.cpp

```diff
@@ -1,6 +1,4 @@
 #include "eliterizer.h"
-#include "ios/lockscreen.h"
```

Comment:
Note to self: check that these removed includes QtCreator identifies as unused are still unused in builds GitHub's CI doesn't do (yet), e.g. iOS and Android.